### PR TITLE
ooc no longer checks if message is ic if round hasn't started

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
what it says on the tin

## Changelog
:cl:
tweak: ooc no longer checks if message is ic if round hasn't started
/:cl:

### CODE REVIEW

File: code/modules/client/verbs/ooc.dm [hunk 1]
```diff
@@ -36,7 +36,7 @@ GLOBAL_VAR_INIT(normal_ooc_colour, "#002eb8")
 
 	msg = emoji_parse(msg)
 
-	if((msg[1] in list(".",";",":","#")) || findtext_char(msg, "say", 1, 5))
+	if(SSticker.HasRoundStarted() && (msg[1] in list(".",";",":","#")) || findtext_char(msg, "say", 1, 5))
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/client/verbs/ooc.dm:39]: http://www.byond.com/docs/ref/#/operator

you messed up the existing check due to order of operations.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
What cobby said

File: code/modules/client/verbs/ooc.dm (hunk 1 above)
Comment (LoserWasTaken) [reply code/modules/client/verbs/ooc.dm:39]: oops i didn't even notice

Comment (LoserWasTaken) [reply code/modules/client/verbs/ooc.dm:39]: i looked at that for an entire minute to notice what was wrong lol

**Pull request merged**